docs/source/howto/run_pwbands.md

```diff
@@ -0,0 +1,133 @@
+(howto-workflows-pw-bands)=
+
+# Calculate a band structure
+
+The `PwBandsWorkChain` is designed to compute electronic band structures for a given structure using Quantum ESPRESSO's `pw.x`.
+It automates the complete workflow, starting from a previously relaxed structure then it performs an SCF calculation, and bands calculation along high-symmetry k-points paths.
+
+|                     |                                                               |
+|---------------------|---------------------------------------------------------------|
+| Workchain class     | {class}`aiida_quantumespresso.workflows.pw.bands.PwBandsWorkChain` |
+| Workchain entry point | ``quantumespresso.pw.bands``                                |
+
+---
+
+## Minimal example: build and submit
+
+```python
+from aiida import orm, load_profile
+from aiida_quantumespresso.workflows.pw.bands import PwBandsWorkChain
+from aiida.engine import submit
+from ase.build import bulk
+
+load_profile()
+
+# Load your pw.x code and structure
+code = orm.load_code('pw@localhost')
+structure = orm.StructureData(ase=bulk('Si', 'diamond', 5.4))
+
+# Get a builder with sensible default parameters from a predefined protocol
+builder = PwBandsWorkChain.get_builder_from_protocol(
+    code=code,
+    structure=structure,
+    protocol="balanced",  # choose from: fast, balanced, stringent
+    options={
+        "account": "your_account", # Change to your account if needed by your HPC provider. Otherwise, remove this line.
+        "queue_name": "debug",
+        "resources": {"num_machines": 1},
+        "max_wallclock_seconds": 1800,
+    }
+)
+
+# Submit the work chain
+workchain_node = submit(builder)
+print(f"Launched {workchain_node.process_label} with PK = {workchain_node.pk}")
+```
+
+The workchain will automatically:
+1. Use SeeKpath to find the primitive cell and generate a high-symmetry k-points path.
+2. Run an SCF calculation to obtain the ground state.
+3. Run a bands calculation along the high-symmetry k-points path.
+
+---
+
+## Customizing the workflow
+
+### Specifying custom k-points path
+
+You can provide your own k-points instead of using SeeKpath:
+
+```python
+from aiida import orm
+
+# Create a custom KpointsData. For example, a list of high-symmetry points:
+kpoints = orm.KpointsData()
+kpoints.set_kpoints([
+    [0.0, 0.0, 0.0],  # Gamma
+    [0.5, 0.0, 0.5],  # X
+    [0.5, 0.25, 0.75],  # W
+    # ... more k-points
+])
+
+builder.bands_kpoints = kpoints
+del builder.bands_kpoints_distance # Remove the distance input if previously set
```

Comment:
Interesting, I've never used `del` in this context. Generally, I don't ever use `del` anywhere. My preference is usually the following:

```suggestion
builder.pop('bands_kpoints_distance', None) # Remove the distance input
```

Note that this will work even if the `bands_kpoints_distance` is not set, whereas the `del` approach will result in a `KeyError`.